services/src/main/java/org/fao/geonet/services/main/GenericController.java

```diff
@@ -77,25 +78,29 @@ public void dispatch(@PathVariable String portal,
         if (forwardedFor != null)
             ip = forwardedFor;
 
-        Log.info(Log.REQUEST, "==========================================================");
+        Logger LOGGER = Log.createLogger(GenericController.class, Log.REQUEST_MARKER);
 
-        Log.info(Log.REQUEST, "HTML Request (from " + ip + ") : " + request.getRequestURI());
-        if (Log.isDebugEnabled(Log.REQUEST)) {
-            Log.debug(Log.REQUEST, "Method       : " + request.getMethod());
-            Log.debug(Log.REQUEST, "Content type : " + request.getContentType());
+
+        LOGGER.info("==========================================================");
+
+        LOGGER.info("HTML Request (from " + ip + ") : " + request.getRequestURI());
+        if (LOGGER.isDebugEnabled()) {
+            LOGGER.debug("Method       : {}",request.getMethod());
+            LOGGER.debug("Content type : {}",request.getContentType());
             // Log.debug(Log.REQUEST, "Context path : "+ req.getContextPath());
             // Log.debug(Log.REQUEST, "Char encoding: "+
             // req.getCharacterEncoding());
-            Log.debug(Log.REQUEST, "Accept       : " + request.getHeader("Accept"));
+            LOGGER.debug( "Accept       : {}",request.getHeader("Accept"));
             // Log.debug(Log.REQUEST, "Server name  : "+ req.getServerName());
             // Log.debug(Log.REQUEST, "Server port  : "+ req.getServerPort());
         }
 
-        if (Log.isDebugEnabled(Log.REQUEST)) {
+        if (LOGGER.isDebugEnabled()) {
             if (httpSession != null) {
-                Log.debug(Log.REQUEST, "Session id is " + httpSession.getId());
+
+                LOGGER.debug( "Session id is {0}",httpSession.getId());
```

Comment:
I'm not sure if should be `{}` instead of `{0}`or if the last is also valid

Reply:
It is a